### ISSUE TITLE
links to 3gpp files now missing in exported GPX

### ISSUE THREAD

**Issue by PontiacCZ** [labexp/osmtracker-android#305]
Export somehow got broken in the latest version. Not only it is a couple of times slower, but the exported GPX does not contain links to audio files.

Snippet of code from a GPX that was exported before the new version came out:

```
	<wpt lat="48.935846" lon="15.422838">
		<ele>356.8</ele>
		<time>2021-02-11T17:08:28Z</time>
		<name><![CDATA[Voice recording]]></name>
		<link href="2021-02-11_18-08-27.3gpp">
			<text>2021-02-11_18-08-27.3gpp</text>
		</link>
		<sat>0</sat>
	</wpt>
```

and the same track exported now:

```
	<wpt lat="48.935846" lon="15.422838">
		<ele>356.8</ele>
		<time>2021-02-11T17:08:28Z</time>
		<name><![CDATA[Voice recording]]></name>
		<sat>0</sat>
	</wpt>
```

The `<link>` tag is missing.

**Comment by jamescr** [comment]
A new version with the patch is on the way of being distributed (v1.0.1). The patch include a rollback on the export function.

**Comment by skorasaurus** [comment]
> A new version with the patch is on the way of being distributed (v1.0.1). The patch include a rollback on the export function.

Thank you for the quick fix; the files are now linked for me in 1.0.1 after I reexported my file :)

**Comment by PontiacCZ** [comment]
Yep, I confirm the issue is fixed with v1.0.1. And the speed is back as well. Thank you for a quick response! :-)
